vite.config.ts

```diff
@@ -1,8 +1,15 @@
 import { defineConfig } from 'vite';
 import tailwindcss from '@tailwindcss/vite';
 import react from '@vitejs/plugin-react';
+import path from 'path';
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react(), tailwindcss()],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src/*'),
+      '@functions': path.resolve(__dirname, '@./functions/*'),
```

Comment:
Additionally we probably wouldn't need an alias for functions within `vite.config.ts` yet because there is no reference to it:

```diff
-    '@functions': path.resolve(__dirname, '@./functions/*'),  
```